test/unit/trigger-method.spec.js

```diff
@@ -186,12 +184,6 @@ describe('trigger event and method name', function() {
       it('should call a method named on{Event}', function() {
         expect(this.methodHandler).to.have.been.calledOnce;
       });
-
-      it('should return the value returned by the on{Event} method', function() {
```

Comment:
I deleted this test because it required spying on `Marionette.triggerMethod`. In the new world of modules, we'd have to split triggerMethod and triggerMethodOn's modules and then stub on the `import` level which is possible but more overhead. Not an essential test and I went w/ killing it